ftw/upgrade/command/help.py

```diff
@@ -24,7 +25,7 @@ def setup_argparser(commands):
     command.set_defaults(func=help_command)
 
     command.add_argument('command', nargs='?',
-                         choices=get_commands(commands.container).keys(),
+                         choices=list(get_commands(commands.container).keys()),
```

Comment:
This was done by the _python-modernize_ tool which applies the [2to3](https://docs.python.org/3/library/2to3.html#2to3fixer-dict) fixers.

Seems they want to be on the safe side in case of argument introspection:
https://github.com/python/cpython/blob/3.8/Lib/lib2to3/fixes/fix_dict.py#L24

I'll keep it that way for now, but I agree that on Python 2 it would be more efficient without the `.keys()` call.